dcpy/connectors/edm/recipes.py

```diff
@@ -194,7 +233,14 @@ def plan_recipe(recipe_path: Path) -> Recipe:
 
     for ds in recipe.inputs.datasets:
         if ds.version is None:
-            if (
+            if ds.version_env_var is not None:
```

Comment:
This is for a case where we're specifying dataset versions via env var? Just want to be crystal clear

Reply:
yep, open to suggestion if there's a more obvious name